Adyen/Analytics/Models/AnalyticsEventInfo.swift

```diff
@@ -35,5 +37,12 @@ public struct AnalyticsEventInfo: AnalyticsEvent {
         case unfocus = "Unfocus"
         case validationError = "ValidationError"
         case rendered = "Rendered"
+        case displayed = "Displayed"
```

Comment:
What is the difference between rendered and displayed?

Reply:
Turns out this is not needed, it's web only so will remove!